Sources/Purchasing/Offering.swift

```diff
@@ -165,6 +173,14 @@ import Foundation
 
 }
 
+extension Offering {
```

Comment:
I'd make this `private` since it doesn't need to be visible outside this file:
```suggestion
private extension Offering {
```